src/core/document.js

```diff
@@ -299,6 +299,8 @@ var Page = (function PageClosure() {
           resources: this.resources,
           normalizeWhitespace,
           combineTextItems,
+          sink,
+          seenStyles: Object.create(null),
```

Comment:
Is there actually any reason to do this, can't you just use `seenStyles = Object.create(null),` in the method signature in `evaluator.js` instead?